base_layer/core/src/consensus/consensus_constants.rs

```diff
@@ -52,6 +55,13 @@ impl ConsensusConstants {
         ConsensusConstants::default()
     }
 
+    /// Use the the current consensus constants with a configurable chain network.
+    pub fn current_with_network(network: Network) -> Self {
+        let mut consensus_constants = ConsensusConstants::current();
```

Comment:
I think this needs to change, but it can change in a next PR. 
The constants will change between the different network types, so we need more than one "default"

Reply:
Yup. Agree

Reply:
I'm leaving this for the next PR.